disruption_generator/cli.py

```diff
@@ -1,18 +1,70 @@
 # -*- coding: utf-8 -*-
 
 """Console script for disruption_generator."""
+import asyncio
+import asyncssh
 import sys
 import click
+import logging
+
+from . import __version__
+from .parsers.experiment_parser import ExperimentParser
+from .listener.alistener import Alistener
+from .trigger.trigger import Trigger
+from os import walk, path
+
+logger = logging.getLogger(__name__)
 
 
 @click.command()
-def main(args=None):
+@click.option(
```

Comment:
It's been implemented as an option since not providing this argument during execution will default to ./experiments (in the context of the execution). We can reconsider removing this and requesting the experiment path as mandatory.